### PR TITLE
add function to update events

### PR DESCRIPTION
**Pull request by MathiasHobler**
pls review the update function

### CODE REVIEW

File: src/components/Detail/Detail.jsx [hunk 1]
```diff
@@ -1,8 +1,17 @@
 import {Bookmark, Close, Delete} from '@mui/icons-material';
+import {NavLink} from 'react-router-dom';
+
+import {useStep, useCreate} from '../../hooks/useForm';
 
 import DetailPOP from './Detail.styled';
 
 const Detail = ({event, back, deleteEvent, bookmark}) => {
```
Comment (stefankorder) [inline src/components/Detail/Detail.jsx:8]: Don't you pass those properties down where you call the Details component in App.js?

File: src/components/Detail/Detail.jsx [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+	const setNewEvent = useCreate(state => state.setNewEvent);
+	const setAction = useStep(state => state.setAction);
+	const setTitle = useStep(state => state.setTitle);
+	setAction('update');
```
Comment (stefankorder) [inline src/components/Detail/Detail.jsx:12]: I think it would be better to put this functions inside a useeffect.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! There are two questions / suggestions 😃

File: src/components/Detail/Detail.jsx (hunk 1 above)
Comment (MathiasHobler) [reply src/components/Detail/Detail.jsx:8]: The App.js route is currently no longer used at all. That was just my first thought this morning. I have removed the route there again

**Pull request merged**